2.4/root/usr/libexec/fix-permissions

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 # Fix permissions on the given directory to allow group read/write of 
 # regular files and execute of directories.
-chgrp -R 0 $1
-chmod -R g+rw $1
+find $1 -exec chgrp -R 0 {} \;
```

Comment:
fyi i updated the jenkins fix-permissions to remove the -R there.  thanks for catching that @php-coder